### DISCUSSION TITLE
Trying to login for first time: "This user account is blocked."

### DISCUSSION THREAD

**Discussion by tlcaputi** [vercel/community#4095] [Account Issue] [answered]
### Question

Hi, I'm new to Vercel -- I'm not sure I've ever logged in before. However, when I try to login with my Github credentials it says "This user account is blocked." Is there a reason that I'm blocked? Can I be unblocked? I can assure you that I am the user I claim to be and that I haven't and wouldn't use Vercel for any malicious purpose -- I just want to check out some of the features.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble logging in, @tlcaputi. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy. That will get you in contact with the team that can help with blocked accounts and login issues.

**Reply by tlcaputi** [reply]
I did not receive an email with more details about the error. I wrote to registration@vercel.com but haven't heard back. Could you have registration@vercel.com write back to me?

**Reply by amyegan** [reply]
Tickets are processed by the order in which they are received. That means it can take a little while to get a response at times when the team is handling a large volume of cases. I know it's frustrating to wait, and we appreciate your patience 🙇 

More info about wait times: https://vercel.com/guides/vercel-support-queue-time#how-vercel-support-works

**Reply by starry-wang** [reply]
my account has the same question, please help me again. my case num : 00159493. 
Thanks!

**Reply by tlcaputi** [reply]
I still haven't received anything from registration@vercel.com, unfortunately.

**Reply by SkyElephone** [reply]
my account has the same question, please help me again. my case num : 00160324.
Thanks!

**Comment by SkyElephone** [comment]
Hi, I'm new to Vercel -- I'm not sure I've ever logged in before. However,
when I try to login with my Github credentials it says "This user account
is blocked.
    Is it the problem with my GitHub account? I don't know.

1. Now I am deploying a chatgpt project on github, named "ChatGPT Next Web",

2. When I click deploy, I will jump to the deploy vercel page and click
"continue with github"

3. There is an error saying 'This user account is blocked.'


Swarnava Sengupta ***@***.***> 于 2023年9月21日周四 下午3:51写道：
